backend/app/routers/status.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+from fastapi.security import HTTPBearer
+
+from app.models.status import Status
+
+router = APIRouter()
+security = HTTPBearer()
+
+
+@router.get("", response_model=Status)
+async def add_thumbnail():
```

Comment:
sorry nitpicking. The function name has thumbnail. Do we want to change to status?
You might also need to run codegen after that